pkgs/applications/science/biology/cnvkit/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchPypi
+, rPackages
+, rWrapper
+, buildPythonPackage
+, biopython
+, numpy
+, scipy
+, pandas
+, matplotlib
+, reportlab
+, pysam
+, future
+, pillow
+, pomegranate
+, pyfaidx
+}:
+
+let
+  cnvR = rWrapper.override {
+    packages = with rPackages; [ DNAcopy ];
+  };
+
+in buildPythonPackage rec {
+  pname = "CNVkit";
+  version = "0.9.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hj8c98s538i0hg5mrz4bw4v07qmcl51rhxq611rj2nglnc9r25y";
+  };
+
+  propagatedBuildInputs = [
+    biopython
+    numpy
+    scipy
+    pandas
+    matplotlib
+    reportlab
+    pyfaidx
+    pysam
+    future
+    pillow
+    cnvR
```

Comment:
How is this supposed to work? The cli is likely going to work because cnvR gets wrapped, but the library won't be able to find cnvR.

Reply:
R is only needed for plotting and I have verified plots do work from the CLI. I'm unsure how to even test cnvlib since I'm ignorant of any dependents. The library excluding plotting functions should still work in the absence of R though. Do you have any suggestions on how to resolve this?

Reply:
I suggest leaving R out of this then. You can document that for plotting one needs to install R.